examples/example.ipynb

```
@@ -0,0 +1,430 @@
+                "    @property\n",
+                "    def author_names(self) -> str:\n",
+                "        \"\"\"All authors' names separated by comma.\"\"\"\n",
+                "        return \", \".join([author.name for author in self.authors.all()])\n",
+                "\n",
+                "\n",
+                "class Library(models.Model):\n",
+                "    name = models.CharField(max_length=100)\n",
+                "    address = models.TextField()\n",
+                "\n",
+                "    class Meta:\n",
+                "        app_label = \"tests\"\n",
```

Ruff (0.11.9)

Module level import not at top of file

(E402)